rocks-db/src/lib.rs

```diff
@@ -549,14 +546,9 @@ impl Storage {
             asset::AssetDynamicDetails::NAME => {
                 let mf = match migration_state {
                     MigrationState::Version(version) => match *version {
-                        CollectionAuthorityMigration::VERSION
-                            ..=ExternalPluginsMigration::VERSION => {
+                        CollectionAuthorityMigration::VERSION => {
```

Comment:
i guess we do not need external_plugins and collection_authority migrations too, as far as we reingest/backfill all data in CompleteAssetDetails and drop all old data here

Reply:
If so we can create separate ticket to drop that stuff as well